evelink/api.py

```diff
@@ -360,11 +360,11 @@ def requests_request(self, full_path, params):
             if params:
                 # POST request
                 _log.debug("POSTing request")
-                r = session.post(full_path, data=params)
+                r = session.post(full_path, data=params, timeout=60)
```

Comment:
It'd be better to have this be a module-level constant - that way if someone wants to change it they can.